.github/workflows/chart-lint-test.yml

```diff
@@ -0,0 +1,49 @@
+name: Chart Lint and Test
+
+on:
+  push:
+    paths:
+      - 'deploy/helm/**'
+    branches:
+      - develop
+  pull_request:
+    paths:
+      - 'deploy/helm/**'
+    branches:
+      - develop
+
+jobs:
+  lint-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v3.4.0
+
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+
+      - name: Set up chart-testing
+        uses: helm/chart-testing-action@v2.0.1
+
+      - name: Run chart-testing (list-changed)
+        id: list-changed
+        run: |
+          changed=$(ct list-changed --config ct.yaml)
+          if [[ -n "$changed" ]]; then
+            echo "::set-output name=changed::true"
+          fi
+
+      - name: Run chart-testing (lint)
+        run: ct lint --config ct.yaml
+
+      - name: Create kind cluster
+        uses: helm/kind-action@v1.0.0
+        if: steps.list-changed.outputs.changed == 'true'
```

Comment:
We can also have `ct install` to deploy the chart and run a helm test in the cluster to check if the deployment was successful.
```
- name: Run chart-testing (install)
  run: ct install --config ct.yaml
```

Reply:
This I haven't added it intentionally because we will need to add helm repo like [here](https://github.com/openebs/dynamic-localpv-provisioner/blob/develop/.github/workflows/chart-lint-test.yml#L51) before we install.

Reply:
Hey @rajaSahil . I have made the changes. Can you have a look.